### DISCUSSION TITLE
How to change the background color of cmdline popup?

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#2126] [Q&A] [answered]
# update

For any one have the same problem. 
Use the setting here. 
```lua
vim.cmd("hi NoiceCmdlinePopupBorderCmdline guibg=#282828 gui=bold")
vim.cmd("hi NoiceCmdlineIconCmdline guibg=#282828 gui=bold")
vim.cmd("hi NoiceCmdlinePopupBorderHelp  guibg=#282828 gui=bold")
vim.cmd("hi NoiceCmdlineIconHelp  guibg=#282828 gui=bold")
```
Related link: https://www.reddit.com/r/neovim/comments/yekq5x/noice_ui_highlights_broken_with_gruvbox/

---
Original question: 

![image](https://github.com/LazyVim/LazyVim/assets/11295355/3dcc641d-e774-4d98-bcb8-970888edfc0f)

I use gruvbox in nvim. Seems the bkg color of the cmd line prompt is not same here. What ui tool used by this prompt, and how to change the color here?  I want replace the black background color with gruvbox main color here

**Comment by dpetka2001** [comment, marked as answer]
`noice` is the plugin that shows this popup cmdline menu. You can see the available highlight groups it uses [here](https://github.com/folke/noice.nvim#-highlight-groups). My guess would be that the highlight group you want to change is the `NoiceCmdlinePrompt`, but you would have to test it out yourself since I've never done this.
